packages/zone.js/test/common/zone.spec.ts

```diff
@@ -343,7 +343,7 @@ describe('Zone', function() {
 
         const ZoneAwarePromise = global.Promise;
         try {
-          global.Promise = WrongPromise;
+          (global as any).Promise = WrongPromise;
```

Comment:
```suggestion
          global.Promise = WrongPromise;
```
I'm not actually sure about this one. How do you think it should be done?
 - [X]  Leave global uncasted as it was before.
 - [ ]  Replace the `any` global cast with a `(NodeJS.Global & typeof globalThis)` one.
 - [ ]  Don't change anything at all.